scripts/tsconfig.json

```diff
@@ -16,6 +16,7 @@
     {"path": "../packages/mysql-schema-print-types"},
     {"path": "../packages/mysql-test"},
     {"path": "../packages/mysql-typed"},
+    {"path": "../packages/node_modules"},
```

Comment:
It looks like you must have `packages/node_modules` in your local repo, which is messing up the code-gen. Please remove this directory and the references to it.